plugins/node/opentelemetry-instrumentation-pg/test/pg.test.ts

```diff
@@ -141,8 +143,21 @@ describe('pg', () => {
     context.setGlobalContextManager(contextManager);
     instrumentation.setTracerProvider(provider);
 
+    executedQueries = [];
     postgres = require('pg');
     client = new postgres.Client(CONFIG);
+
+    // We need to track the actual queries that get executed to assert
+    // that the sqlcommenter logic works
+    const privateClient: {
+      queryQueue: (pg.Query & { text?: string })[];
+    } = client as any;
+    const originalPush = privateClient.queryQueue.push;
+    privateClient.queryQueue.push = (...items) => {
+      executedQueries.push(...items);
+      return originalPush.apply(privateClient.queryQueue, items);
+    };
+
```

Comment:
nit: A possible alternative to this, which is a bit cleaner and idiomatic, is to use [sinon spy](https://sinonjs.org/releases/v14/spies/)

I think you should be able to do something like:
```js
  beforeEach(() => {
    // other code
    sandbox.spy((client as any).queryQueue, 'push');
  });
```

and then in your assertions, do something like
```js
const executedQueries = (client as any).queryQueue.push.getCall(0).args
```

You will need to also
```js
  afterEach(() => {
    sandbox.restore();
  });
```

This is optional